__tests__/app.test.js

```diff
@@ -460,4 +460,15 @@ describe("DELETE: /api/comments/:comment_id", () => {
         expect(body.msg).toBe("Bad request");
       });
   });
+}); //
+
+describe("GET:  /api", () => {
+  test("200: should responds with JSON object with describing all available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body.endpoints).toBe("object");
```

Comment:
This test won't tell you enough about the object you are getting back. Think about the information within, keys etc. and try to form a more rigid test, as so far an array would pass here.